lib/bolt/applicator.rb

```diff
@@ -185,6 +187,14 @@ def validate_hiera_config(hiera_config)
       end
     end
 
+    def get_trusted_external(certname)
+      Puppet[:trusted_external_command] = @trusted_external
```

Comment:
I'm slightly concerned about changing puppet settings here. Maybe we should do this in PAL?

Reply:
We already set some Puppet settings in the catalog.rb - we could also pass the trusted external command through and set this there since it's executed in a separate process. Is your concern that this will pollute an existing Puppet environment?

Reply:
Updated - I gave it it's own class function in PAL to avoid making a new Pal, and because I didn't think there were any existing functions that seemed right for it to belong in.